docs/user-guide/address-security-requirements.md

```diff
@@ -0,0 +1,73 @@
+| Cross memory server (ZIS)                                  | CSFSERV                                 | `Multiple`                  | READ   | Generate symmetric keys using ICSF that is used by [Zowe Desktop cookies](./configure-zos-system.md#configure-an-icsf-cryptographic-services-environment). | The list of IDs to enable include `CSF1TRD` , `CSF1TRC` , `CSF1SKE` , `CSF1SKD`. The full list of IDs is described in the z/OS Cryptographic Services user guide for your z/OS release level: [2.2](https://www.ibm.com/docs/en/zos/2.2.0?topic=ssl-racf-csfserv-resource-requirements), [2.3](https://www.ibm.com/docs/en/zos/2.3.0?topic=ssl-racf-csfserv-resource-requirements), [2.4](https://www.ibm.com/docs/en/zos/2.4.0?topic=ssl-racf-csfserv-resource-requirements) and [2.5](https://www.ibm.com/docs/en/zos/2.5.0?topic=ssl-racf-csfserv-resource-requirements). |
+| Cross memory server (ZIS)                                  | FACILITY                                | `ZWES.IS`                   | READ   | To allow Zowe ZWESLSTC processes to access the Zowe ZIS cross memory server.  | Run the command to permit Zowe main server to use ZIS cross memory server.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| ZSS (required for API ML certificate and identity mapping) | FACILITY                                | `BPX.SERVER` + `BPX.DAEMON` | UPDATE | Allows Zowe to run code on behalf of the API requester's TSO user ID. For more information, see [Security Environment Switching](./configure-zos-system.md#configure-security-environment-switching). | Run the commands to permit the Zowe main server to create a user's security environment |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
Simplify to "ZSS". There are various reasons why you'd be required to use any component. This column should not be bloated with all those details.

Context:
This row is required by ZSS, a default component. ZSS can exist with or without API ML. Saying that ZSS is required by API ML in certain circumstances is a detail seems like something that instead should be present in the API ML certificate & identity mapping documentation.

Reply:
Keep in mind there's more dependencies, so it could be quite a complex table if you want to fit it all here.

Reply:
I agree. This is one of the biggest configuration challenges that I am currently seeing. Admins are not aware of dependencies between components. I tried to address this feedback at least partially here.